PythonPonies/bangazon-api
# Ls paymenttypemodel

# Description
Add description about what this pull request covers.

This adds the model, view, and serializer for the payment type table, incorporating hyperlinks to the payment type list and detail views. 

## Number of Fixes
Type of number of fixes included in this PR.

This has been deconflicted with the two previous merges and displays hyperlinks and recognizes foreign key relationships to the other two tables, user and order. 

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#53 #30  #17 #29 #45 #36 #10  #1 

## Problem to Solve
Describe the problem(s) this pull request solves.
Creates payment type table and links via FK relationship to all other existing tables.

## Proposed Changes
Describe the proposed changes.
None.

## Expected Behavior
Describe the expected behavior.
The payment type should be visible on the root api page with links that, when clicked, route to the payment list view, which has no data, but should link to payment detail view when sample data is added and clicked on.

## Steps to Test Solution

1. Fetch the branch, run the server, navigate to localhost:8000, click the /payment link and verify that the behavior above replicates. 

## Testing
There are no unit tests. 

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
